packages/api/src/routes/medical/internal-docs.ts

```diff
@@ -179,6 +180,7 @@ router.post(
   asyncHandler(async (req: Request, res: Response) => {
     const cxId = getUUIDFrom("query", req, "cxId").orFail();
     const patientId = getFrom("query").orFail("patientId", req);
+    const source = getFrom("query").orFail("source", req);
```

Comment:
Since this is a limited list of values, I suggest we have a schema. The current approach would result in using CQ if we send a typo on `source`, for example.